containers/hugging-face-inference/terraform/providers.tf

```diff
@@ -0,0 +1,16 @@
+provider "scaleway" {
+  region     = var.region
+  access_key = var.access_key
+  secret_key = var.secret_key
+  project_id = var.project_id
```

Comment:
nit: IMO this is unnecessary, the default behavior of the provider is to use your config file or the environment to get its configuration, so I would leave it blank